src/components/Contacts/ContactDetails/ContactTaskRow/TaskCompleteButton/TaskCompleteButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, styled, Theme } from '@material-ui/core';
+import { Check } from '@material-ui/icons';
+import React from 'react';
+
+const TaskRowWrap = styled(Button)(
+  ({ theme, isComplete }: { theme: Theme; isComplete: boolean }) => ({
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: isComplete ? theme.palette.mpdxGreen.main : 'transparent',
+    height: 24,
+    width: 55,
+    borderRadius: 26,
+    border: '2px solid #00CA99',
+    margin: theme.spacing(2),
+  }),
+);
+
+const TaskCheckIcon = styled(Check)(
+  ({ theme, isComplete }: { theme: Theme; isComplete: boolean }) => ({
+    color: isComplete
+      ? theme.palette.common.white
+      : theme.palette.mpdxGreen.main,
+  }),
+);
+
+interface TaskCompleteButtonProps {
+  isComplete: boolean;
+  onClick: () => void;
+}
+
+export const TaskCompleteButton: React.FC<TaskCompleteButtonProps> = ({
+  isComplete,
+  onClick,
+}) => {
+  return (
+    <TaskRowWrap isComplete={isComplete} onClick={() => onClick()}>
+      <TaskCheckIcon data-testid="checkIcon" isComplete={isComplete} />
```

Comment:
```suggestion
      <TaskCheckIcon titleAccess={t('Check Icon')} isComplete={isComplete} />
```
would be better. You can do `findByRole('img', { name: 'Check Icon' })` to get at it.